.env.sample

```diff
@@ -0,0 +1,11 @@
+/** SAMPLE ENV FILE **/
+
+/*SPECIFY NODE ENVIRONMENT TO RUN IN
+  USE TEST FOR LOCAL DEVELOPMENT
+*/
+NODE_ENV=test
+
+/*SPECIFY DB TO USE
+  ENTER THE LOCAL DB URL HERE
+*/
+DB=mongodb://localhost:27017/cfh
```

Comment:
@andela-odaniel This is good, please change the DB variable to something more explanatory, DB_URL looks good